cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -0,0 +1,304 @@
+import org.neo4j.kernel.api.KernelTransaction.Type
+import org.neo4j.kernel.api.security.AnonymousContext
+
+import scala.collection.JavaConverters._
+
+class AliasesTest extends DocumentingTest with QueryStatisticsTestSupport {
+  override def outputPath = "target/docs/dev/ql/administration/"
+
+  override def doc: Document = new DocBuilder {
+    doc("Alias management", "alias-management")
+    database("system")
+    initQueries(
+      "CREATE DATABASE `movies`",
+      "CREATE ALIAS `films` FOR DATABASE `movies`",
+      "CREATE ALIAS `motion pictures` FOR DATABASE `movies`",
+      "CREATE DATABASE `northwind-graph-2020`",
+      "CREATE DATABASE `northwind-graph-2021`",
+      """CREATE ALIAS `movie scripts` FOR DATABASE `scripts` AT "neo4j://localhost:7687" USER alice PASSWORD "password" DRIVER {
+        |    ssl_enabled: true,
+        |    connection_timeout: duration({seconds: 5}),
+        |    connection_max_lifetime: duration({hours: 1}),
+        |    connection_pool_acquisition_timeout: duration({minutes: 1}),
+        |    connection_pool_idle_test: duration({minutes: 2}),
+        |    connection_pool_max_size: 10,
+        |    logging_level: 'info'
+        |}""".stripMargin
+    )
+    synopsis("This chapter explains how to use Cypher to manage database aliases in Neo4j.")
+    p(
+      """There are two kinds of aliases, local database aliases and remote database aliases.
+        |A local database alias can only target a database within the same DBMS.
+        |A remote alias may target a database from another Neo4j DBMS.
+        |When a query is run against an alias, it will be redirected to the target database.
+        |The home database for users can be set to an alias, which will be resolved to the target database on use.
+        """.stripMargin)
+    p(
+      """A local alias can be used in all other Cypher commands in place of the target database.
+        |Please note that the local alias will be resolved while executing the command.
+        |Privileges are defined on the database, and not the local alias.
+        """.stripMargin)
+    p(
+      """A remote alias can be used for connecting to a database of a remote Neo4j DBMS, use clauses, setting a user's home database and defining the access privileges to the remote database.
+        | Remote aliases requires configuration to safely connect to the remote target, which is described in <<operations-manual#manage-remote-aliases, Connecting remote databases>>.
+        """.stripMargin)
+    p(
+      """Aliases can be created and managed using a set of Cypher administration commands executed against the `system` database.
+        |When connected to the DBMS over bolt, administration commands are automatically routed to the `system` database.""".stripMargin)
+    p("include::alias-command-syntax.asciidoc[]")
+    p("include::alias-driver-settings-table.asciidoc[]")
+    section("Listing database aliases", "alias-management-show-alias") {
```

Comment:
```suggestion
    section("Listing database aliases", "alias-management-show-alias, role = "enterprise-edition") {
```

Looking at the Neo4j code, I think this should also be marked as enterprise, right?